misoc/cores/sdram_settings.py

```diff
@@ -87,6 +87,21 @@ class MT48LC4M16(SDRAMModule):
     tRFC  = 66
 
 
+class MT48LC16M16(SDRAMModule):
+    memtype = "SDR"
+    # geometry
+    nbanks = 4
+    nrows  = 8192
+    ncols  = 512
+    # timings (-75 speedgrade)
+    tRP   = 20
+    tRCD  = 20
+    tWR   = 15
+    tWTR  = 2 #?
+    tREFI = 64*1000*1000/8192 # what's this?
```

Comment:
Refresh interval. Search for tREFI in e.g. http://utaharch.blogspot.com/2013/11/a-dram-refresh-tutorial.html for an explanation.